### ISSUE TITLE
DoorLock ConfigurationSet ERROR

### ISSUE THREAD

**Issue by ripper121** [Z-Wave-Me/ExpertUI#238]
I cant use the DoorLock ConfigurationSet.
This is the actual command:
devices[8].instances[0].commandClasses[98].ConfigurationSet(1,1) <- ends in a Error
devices[8].instances[0].commandClasses[98].ConfigurationSet(1,0,01,1) <- this is how it should be

**Comment by PoltoS** [comment]
The prototype is zway_cc_door_lock_configuration_set(ZWay zway, ZWBYTE node_id, ZWBYTE instance_id, ZWBYTE opType, ZWBYTE outsideState, ZWBYTE insideState, ZWBYTE lockMin, ZWBYTE lockSec),

so correct is with 5 params!

**Comment by PoltoS** [comment]
69af9e0420b927b1c3a95be2de73077acf4f4153

**Comment by ripper121** [comment]
Yeah missed a comma, but still the DoorLock ConfigurationSet Button on the Expert UI is not working.

**Comment by PoltoS** [comment]
have you applied the changes proposed?

**Comment by ripper121** [comment]
![2017-02-17_1551](https://cloud.githubusercontent.com/assets/11836272/23069784/ff999b38-f528-11e6-8983-ca0a1b76bcd1.png)

This Button is not Working!

**Comment by PoltoS** [comment]
You have not applied the change I've shown. It is not in release yet, it is in git only. So please apply it manually to pyzw_zwave_ui.js
